### PR TITLE
[tinycon] update types

### PR DESCRIPTION
**Pull request by jaulz**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
- `setBubble` also accepts strings or null values, see: https://github.com/tommoor/tinycon/blob/master/tinycon.js#L271
- `fallback` also accepts `force`, see: https://github.com/tommoor/tinycon/blob/83ed386e367d0bb6e27b496ec3b6240ce43f6a27/tinycon.js#L115

### CODE REVIEW

File: types/tinycon/index.d.ts [hunk 1]
```diff
@@ -3,15 +3,15 @@
 // Definitions by: Daniel Waxweiler <https://github.com/dwaxweiler>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function setBubble(count: number): void;
+export function setBubble(count: number|string|null): void;
```
Comment (dwaxweiler) [inline types/tinycon/index.d.ts]: Please add spaces between the types and bars to be consistent.

**Review by dwaxweiler — CHANGES_REQUESTED** [review]
Your changes look good. Just add a few spaces as I have suggested in my comment and some tests to the test file. In addition, you can add your name to the authors in the header (see other packages for inspiration). :)

File: types/tinycon/index.d.ts (hunk 1 above)
Comment (jaulz) [reply types/tinycon/index.d.ts]: Done.

**Comment by jaulz** [comment]
Thanks a lot for your quick feedback! 😊 I just added all your suggested changes.

**Review by dwaxweiler — APPROVED** [review]
:+1: Awesome! Thanks for making the changes, @jaulz!

**Pull request merged**